Models/Base/MetaData/POCO/Id.cs

```diff
@@ -0,0 +1,9 @@
+using System.Text.Json.Serialization;
+
+namespace Backend.Models.Base.MetaData.POCO
+{
+    public class Id
+    {
+        [JsonPropertyName("$oid")] public string Oid { get; set; }
```

Comment:
What does these `$name` property names do?

Reply:
I think thats just how they are written in the collection
```json
   "_id":{
      "$oid":"6038ea4bdc2fdbd2bb7ad8db"
   },
```

Reply:
Oh. Alright!